lib/optionsFromArguments.js

```diff
@@ -93,6 +95,8 @@ const optionsFromArguments = function optionsFromArguments(args) {
   options = fromEnvironmentOrDefault(options, 'baseUrlDirect', 'S3_BASE_URL_DIRECT', false);
   options = fromEnvironmentOrDefault(options, 'signatureVersion', 'S3_SIGNATURE_VERSION', 'v4');
   options = fromEnvironmentOrDefault(options, 'globalCacheControl', 'S3_GLOBAL_CACHE_CONTROL', null);
+  options = fromEnvironmentOrDefault(options, 'presignedUrl', 'S3_PRESIGNED_URL', false);
+  options = fromEnvironmentOrDefault(options, 'presignedUrlExpires', 'S3_PRESIGNED_URL_EXPIRES', 900);
```

Comment:
How did you get to 900? Is this a random value you chose?

Reply:
i think the previous PR owner figured 15 mins was a good default value, should this be changed?

Reply:
Not sure, do we have to provide a default value at all?

Reply:
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getSignedUrl-property

Oh it does look like s3's `getSignedURL` defaults `Expires` to 900 seconds

> Options Hash (params):
>
> Expires (Integer) — default: 900 — the number of seconds to expire the pre-signed URL operation in. Defaults to 15 minutes.

Reply:
So we don't have to set a default value, as the S3 SDK will set that, right?